episodes/files/environment.yml

```diff
@@ -0,0 +1,11 @@
+name: dc-image
+channels:
+  - conda-forge
+dependencies:
+  - python>=3.11
+  - jupyterlab
+  - numpy
+  - matplotlib
+  - pandas
```

Comment:
```suggestion
```
I don't think we use pandas in this lesson? We mention it in the 'image basics' episode, but don't actually use it.